### PR TITLE
Update Context IRIs

### PR DESCRIPTION
**Pull request by jhaag75**
- Update to Context IRIs
- Update to conformsTo IRIs

### CODE REVIEW

File: profiles/acrossx.jsonld [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "@context": "https://w3id.org/xapi/profiles/context",
   "id": "https://w3id.org/xapi/acrossx",
   "type": "Profile",
-  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```
Comment (fugu13) [inline profiles/acrossx.jsonld:5]: This is not the correct URI. Per the spec, conformsTo must be:  https://w3id.org/xapi/profiles#1.0

File: profiles/acrossx.jsonld [hunk 2]
```diff
@@ -13,9 +13,8 @@
   },
   "versions": [
     {
-      "id": "https://w3id.org/xapi/acrossx/v1",
-      "generatedAtTime": "2017-07-14T11:35:00Z",
-      "wasRevisionOf": ["https://w3id.org/xapi/acrossx"]
+      "id": "https://w3id.org/xapi/acrossx",
```
Comment (fugu13) [inline profiles/acrossx.jsonld:16]: You can't use the same identifier for the profile as a whole and a profile version.

File: profiles/scorm.jsonld [hunk 3]
```diff
@@ -1,18 +1,15 @@
 {
-    "id": "https://w3id.org/xapi/scorm#profile",
+   "@context": "https://w3id.org/xapi/profiles/context",
+    "id": "https://w3id.org/xapi/scorm",
```
Comment (fugu13) [inline profiles/scorm.jsonld:3]: You can't use the same id for the profile as a whole and a profile version

**Comment by jhaag75** [comment]
Somehow I missed this! Thanks Russell. Will correct and submit a PR.

File: profiles/seriousgames.jsonld [hunk 4]
```diff
@@ -15,9 +15,8 @@
   },
   "versions": [
     {
-      "id": "https://w3id.org/xapi/seriousgames/v1",
-      "generatedAtTime": "2017-07-14T10:00:00Z",
-      "wasRevisionOf": ["https://w3id.org/xapi/seriousgames"]
+      "id": "https://w3id.org/xapi/seriousgames",
```
Comment (fugu13) [inline profiles/seriousgames.jsonld:18]: Identifier duplication again

File: profiles/video.jsonld [hunk 5]
```diff
@@ -16,9 +16,8 @@
   },
   "versions": [
     {
-      "id": "https://w3id.org/xapi/video/v1",
-      "generatedAtTime": "2017-06-29T10:45:00Z",
-      "wasRevisionOf": ["https://w3id.org/xapi/video"]
+      "id": "https://w3id.org/xapi/video",
```
Comment (fugu13) [inline profiles/video.jsonld:19]: identifier duplication again

File: profiles/video.jsonld [hunk 6]
```diff
@@ -1,8 +1,8 @@
 {
-  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "@context": "https://w3id.org/xapi/profiles/context",
   "id": "https://w3id.org/xapi/video",
   "type": "Profile",
-  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```
Comment (fugu13) [inline profiles/video.jsonld:5]: needs to be URI from spec

File: profiles/scorm.jsonld [hunk 7]
```diff
@@ -1,18 +1,15 @@
...
     "prefLabel": {
         "en": "Experience API SCORM Profile"
     },
+    "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```
Comment (fugu13) [inline profiles/scorm.jsonld:7]: needs to be URI from spec

File: profiles/cmi5.jsonld [hunk 8]
```diff
@@ -1,7 +1,8 @@
 {
+   "@context": "https://w3id.org/xapi/profiles/context",
     "id": "https://w3id.org/xapi/cmi5",
     "type": "Profile",
-    "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+    "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```
Comment (fugu13) [inline profiles/cmi5.jsonld:5]: needs to be URI from spec

File: profiles/adb.jsonld [hunk 9]
```diff
@@ -1,8 +1,8 @@
 {
-  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "@context": "https://w3id.org/xapi/profiles/context",
   "id": "https://w3id.org/xapi/adb",
   "type": "Profile",
-  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "conformsTo": "https://github.com/adlnet/xapi-profiles/",
```
Comment (fugu13) [inline profiles/adb.jsonld:5]: needs to be URI from spec

**Comment by fugu13** [comment]
The .DS_Store this PR added needs to be removed.

**Pull request merged**